docs/janssen-server/fido/trust-diagnostics.md

```
@@ -0,0 +1,91 @@
+    ```
+
+=== "Config API (fido2 plugin)"
+
+    ```
+    GET /fido2/trust/attestation/config
+    ```
+
+    Served under the Config API base path. Requires one of the
+    `https://jans.io/oauth/config/fido2.readonly`, `fido2.write`, `fido2.admin` or `read-all` scopes.
+
+```json
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)